.github/maven.settings.xml

```diff
@@ -42,24 +50,32 @@
             <password>${env.NEXUS_PASSWORD}</password>
         </server>
         <server>
-            <id>jahia-enterprise-releases</id>
+            <id>jahia-snapshots</id>
+            <username>${env.NEXUS_USERNAME}</username>
+            <password>${env.NEXUS_PASSWORD}</password>
+        </server>
+        <server>
+            <id>jahia-internal</id>
             <username>${env.NEXUS_USERNAME}</username>
             <password>${env.NEXUS_PASSWORD}</password>
         </server>
         <server>
-            <id>jahia-enterprise-snapshots</id>
+            <id>oauth-snapshots</id>
```

Comment:
Please look at the settings.xml file from master (previously used in CircleCI) and import the same parameters